io.gitlab.librewolf-community.desktop

```diff
@@ -0,0 +1,25 @@
+[Desktop Entry]
```

Comment:
Any reason why this one is not upstream?

Reply:
That… is actually a good question – it's pretty much a duplicate of the upstream file which I just never thought of removing in the manual flatpak repo we've used up until now.
I'll get that sorted ^^

Reply:
It's not an exact duplicate; I'd first have to check if a more generic .desktop file would cause any issues upstream before changing this.

Would this block the PR (as in: should I try to get this sorted right away, or can it wait until after it's submitted)?

Reply:
No, it can be fixed later on.